checklists/apim_checklist.en.json

```diff
@@ -419,6 +425,7 @@
             "text": "Use managed identities to authenticate to other Azure resources whenever possible",
             "guid": "791abd8b-7706-4e31-9569-afefde724be3",
             "severity": "Medium",
+            "graph": "resources | where type == "microsoft.apimanagement/service" | extend compliant = (isnotnull(identity)) | distinct id, compliant",
```

Comment:
Double quotes in the query string break the JSON syntax, please use single quotes